pkg/catalogv2/helmop/operation.go

```diff
@@ -678,6 +723,8 @@ func (s *Operations) getChartCommand(namespace, name, chartName, chartVersion st
 	return c, nil
 }
 
+// getInstallCommand receives the repository namespace and name and body of the request
```

Comment:
// getInstallCommand receives the repository namespace and name and body of the request

// getInstallCommand receives the repository namespace, name, and body of the request